### PR TITLE
chore: add default auto-label and blunderbuss configs

### PR DESCRIPTION
**Pull request by chingor13**
Towards b/301957852 to ensure python repos are using blunderbuss to assign to maintainers.

### CODE REVIEW

**Review by bcoe — APPROVED** [review]

File: tests/test_python_library.py [hunk 1]
```diff
@@ -148,6 +149,71 @@ def test_library_noxfile(template_kwargs, expected_text):
...
+    t = templates.Templates(PYTHON_LIBRARY / ".github")
+    result = t.render(
+        "CODEOWNERS",
+        metadata={"repo": {"codeowner_team": "googleapis/foo"}},
+    ).read_text()
+    assert "*     @googleapis/yoshi-python googleapis/foo" in result
+    assert "/samples/   @googleapis/python-samples-reviewers googleapis/foo" in result
+
+
+def test_library_codeowners():
```
Comment (parthea) [inline tests/test_python_library.py:162]: `test_library_codeowners` exists at line 152

```
tests/test_python_library.py:162:1: F811 redefinition of unused 'test_library_codeowners' from line 152
nox > Command flake8 synthtool tests failed with exit code 1
nox > Session lint-3.8 failed.
```

File: synthtool/gcp/templates/python_library/.github/blunderbuss.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Blunderbuss config
+# 
+# This file controls who assigned for pull requests and issues
+# Note: This file is autogenerated. To make changes to the codeowner team, please update .repo-metadata.json.
+{% if metadata['repo']['codeowner_team']|length -%}
+assign_issues:
+  - googleapis/yoshi-python
```
Comment (parthea) [inline synthtool/gcp/templates/python_library/.github/blunderbuss.yml:7]:
```suggestion
```

**Review by parthea — APPROVED** [review]
Added suggestion to remove `yoshi-python` if there is a `codeowner_team` defined.

**Review by parthea — APPROVED** [review]

**Review by m-strzelczyk — APPROVED** [review]

**Comment by parthea** [comment]
Will merge on Monday

**Review by leahecole — APPROVED** [review]

**Pull request merged**